pinot-common/src/main/java/org/apache/pinot/pql/parsers/PinotQuery2BrokerRequestConverter.java

```diff
@@ -65,7 +61,8 @@ public BrokerRequest convert(PinotQuery pinotQuery) {
 
     //Query Type
     QueryType queryType = new QueryType();
-    if (brokerRequest.getAggregationsInfo() != null && brokerRequest.getAggregationsInfo().size() > 0) {
+    if (brokerRequest.getAggregationsInfo() != null
```

Comment:
Not sure if the original code or the new code isn't complaint with Pinot sytling.

Reply:
I am using the pinot style and using IntelliJ. Will check again.

Reply:
fixed